test/cets_test_helper.erl

```diff
@@ -0,0 +1,37 @@
+-module(cets_test_helper).
+-include_lib("eunit/include/eunit.hrl").
+
+-export([
+    get_disco_timestamp/3,
```

Comment:
This could be moved to the disco_SUITE along with `wait_for_disco_timestamp_to_be_updated` from the `cets_test_wait` helper, as it's the only place that uses this outside of the disco_SUITE... but I'm not really sure if moving the wait helper to the suite is a good idea, just some food for thought.